extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -572,17 +596,25 @@ export class ConsentUI {
   resetIframe_() {
     const {classList} = this.parent_;
 
-    // Remove the iframe active to go back to our normal height
-    classList.remove(consentUiClasses.iframeActive);
+    // Remove the iframe active (lightbox) to go back to our normal height
+    classList.remove(
+      this.lightboxEnabled_
```

Comment:
Could this be the same classname, and then differentiate by using a compound selector (e.g. `classA.classB`) for this case? This is a presentation detail that's spilling into logic, making it somewhat fragile.

Reply:
Discussed offline, will defer to using compound selector for clarify and simplicity.